packages/react/src/themes/teams/components/RadioGroup/radioGroupItemStyles.ts

```diff
@@ -64,6 +64,7 @@ const radioStyles: ComponentSlotStylesInput<
     // overrides from icon styles
     backgroundColor: 'transparent',
     boxShadow: 'none',
+    boxSizing: 'border-box',
```

Comment:
Shouldn't this be applied by default?? We are using only svg icons and they already have border-box

Reply:
Based on offline discussion we decided to add a new icon and use it.